media/Config/importConfig.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const oneImportTools = function(data, importOpt, tool, idx, defaultImportObject) {
```

Comment:
As usual JavaScript does, all the parameters doesn't express what the types are and it's a bit hard to understand the codes. Could you please add some comments, in the source code, with doxygen format, that explains each arguments? This is optional but will help others and future you when memory gets fuzzy
```
/**
 * @brief oneImportTools will ...
 * @param data 2D array that holds...
 * @importOpt import options of string type
 * @tool Integer index of ...
 * ...
 */
```

Reply:
My suggestion was wrong ... -_-;;; sorry about this.

Reply:
add param is wrong?

Reply:
it should be like https://github.com/Samsung/ONE-vscode/pull/182#discussion_r718211707 :)